### DISCUSSION TITLE
Please enable the Anonymous Sign-in method, or Authenticate the Client before issuing a request

### DISCUSSION THREAD

**Discussion by dgsdestinygamestudios** [GodotNuts/GodotFirebase#366] [Q&A] [answered]
When I try to sign in, the plugin throws this error:

Please enable the Anonymous Sign-in method, or Authenticate the Client before issuing a request

I'm using signup_with_email_and_password

I've done same things as another project of mine, and I don't get this error there.

**Comment by WolfgangSenff** [comment, marked as answer]
Sounds like you haven't actually turned on the authentication method in the Firebase console. This is one of the most tested features in our plugin though and no one else has had issues, so I'm going to go ahead and close this and recommend you turn on the correct auth method, and make sure you authenticate before making any requests.

**Reply by dgsdestinygamestudios** [reply]
I have enabled signing in with email and password. As I said, I've done same things as another project of mine, and I don't get this error there. Same setup, did same things, but getting this error. Deleted the project and started from scratch, still getting the same error.

Edit: If I call the same function at the start of the scene, it does not throw the error.

Edit 2: Now all of a sudden, everything works as intended. Thank you for everything.
